### DISCUSSION TITLE
PGRST301 error in FlutterFlow

### DISCUSSION THREAD

**Discussion by janvanman** [supabase/supabase#22370] [Questions]
Hi everyone,

I'm trying to get records from Supabase from within FlutterFlow and I'm getting this error:

{
    "code": "PGRST301",
    "details": null,
    "hint": null,
    "message": "JWSError (CompactDecodeError Invalid number of parts: Expected 3 parts; got 1)"
}

The url I'm using is : 

https://[projectref].supabase.co/rest/v1/funcGetUsers?vuserid=z1R4nVHS6XViPdshAyVd4H1f0KG2&vlocation=US

with these headers : 

![supabase](https://github.com/supabase/supabase/assets/10946243/f4ed57bb-672b-49cb-81c0-1f99f1018843)

The function I'm calling in Supabase is working fine in Supabase.  I understand that the JWT has three parts but I'm not sure why the API is only seeing one.  I've copied the API key from the section below:

![anonkey](https://github.com/supabase/supabase/assets/10946243/93fbc889-f473-4954-a163-f6feb766e964)

In Postman I get a "401 Unauthorized" return with the same error response as above.  

What can I try or is there something I'm missing?

Thanks
J

**Comment by GaryAustin1** [comment]
Where are you getting the authorization header jwt?

**Reply by janvanman** [reply]
I copied it from Project API keys

**Reply by GaryAustin1** [reply]
If that is supposed to be anon jwt then odd the beginning part is different.
![image](https://github.com/supabase/supabase/assets/54564956/f211fa97-967b-45ab-ae6b-9250c3c29099)
